test/mavsdk_tests/autopilot_tester_follow_me.cpp

```diff
@@ -186,21 +186,24 @@ void AutopilotTesterFollowMe::connect(const std::string uri)
 }
 
 
-void AutopilotTesterFollowMe::straight_line_test(const float altitude_m, const bool stream_velocity)
+void AutopilotTesterFollowMe::straight_line_test(const bool stream_velocity)
 {
-	const unsigned location_update_rate = 1;
-	const float position_tolerance = 5.0f;
+	// CONFIGURATION for the test
+	const unsigned location_update_rate = 1; // [Hz] How often the GPS location update samples are generated
+	const float position_error_tolerance = 11.0f; // [m] Position error tolerance in both X and Y direction
+	const float follow_me_height_setting = 10.0f; // [m] Height above home position where the Drone will follow from
```

Comment:
What do you mean by this? What takeoff altitude?

Reply:
Well, you deleted the first argument from the function. Originally it was 
```
void AutopilotTesterFollowMe::straight_line_test(const float altitude_m, const bool stream_velocity)
```

and now it's
```
void AutopilotTesterFollowMe::straight_line_test(const bool stream_velocity)
```

and I don't see an advantage :)

Reply:
I thought that there's no good reason to have the `altitude_m` as an argument to the function, since we are just doing a test, and I think it's fine to handle altitude setpoint inside the function. What do you think @potaito ?